sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsModelFactory.cs

```diff
@@ -703,7 +704,8 @@ internal static LinkedEntityMatch LinkedEntityMatch(double confidenceScore = def
                 recognizePiiEntitiesActionResults.ToList(),
                 recognizeLinkedEntitiesActionResults.ToList(),
                 analyzeSentimentActionResults.ToList(),
-                extractSummaryActionResults.ToList());
+                extractSummaryActionResults.ToList(),
+                new List<RecognizeCustomEntitiesActionResult>());
```

Comment:
you could add a parameter to allow the user to set the `List<RecognizeCustomEntitiesActionResult>` so you can pass that here. Similar to `extractSummaryActionResults`